CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
 sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
 echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
 
-sudo apt update && sudo apt install bazel clang-format g++-12
+sudo apt update && sudo apt install bazel=6.4.0 clang-format g++-12
```

Comment:
might be easier to just use [bazelisk](https://github.com/bazelbuild/bazelisk/blob/master/README.md#installation)